utils/react-native/README.md

```diff
@@ -0,0 +1,3 @@
+# @ma-shop/react-native
+
+This package holds several utils specific to react
```

Comment:
This is nitpicking, but update this line to end with "react native" since technically there could be react only utils in the future